### ISSUE TITLE
More complete documentation is needed

### ISSUE THREAD

**Issue by schutze** [hhurz/tableExport.jquery.plugin#190]
This plugin is a major improvement over the defunct kayalshri/tableExport.jquery plugin but the documentation seems very incomplete as only some of the options are explained in the readme.
There should be more extensive documentation that covers every option as well as sample code.

**Comment by hhurz** [comment]
You are right to complain the lack of sufficient documentation. The only excuse i can offer is, that I forked that abandoned plugin and adapted it to my needs (and to some others on request). I had never time and aspiration to make a complete package out of it.
